deployment/provision_vagrant_vm.sh

```diff
@@ -36,21 +36,27 @@ service redis-server restart
 # install apache
 apt-get -q install -y apache2 apache2-dev libapache2-mod-wsgi-py3
 
-# With docker for mac, root will own the mount point (uid=0). chmod does not touch the host file system in these cases.
-OWNER=$(stat -c %U "$MOUNTPOINT/evap")
-if [ "$OWNER" == "root" ]; then chown -R 1042 "$MOUNTPOINT"; fi
+# link the mounted evap folder from the home directory:
+OWNER=$(stat -c %u "$MOUNTPOINT/evap")
+if [ "$OWNER" != 1042 ]; then
+  #  if, for any reason, vagrant failed to mount evap under the correct uid, use bindfs to link it to /opt/evap using the wanted user id
+  #  If you are using docker: Make sure /dev/fuse of your host is forwarded into the container
+  apt-get -q install -y bindfs
+  mkdir -p "$REPO_FOLDER"
+  bindfs --map="$OWNER/1042:@$OWNER/@1042" "$MOUNTPOINT" "$REPO_FOLDER" || exit 1
```

Comment:
Coming back to https://github.com/e-valuation/EvaP/pull/2105#discussion_r1462468031:
For me, with a (I'd say) pretty standard Ubuntu 22.04 installation, I'll need to have the additional `--device=/dev/fuse` docker argument in my Vagrantfile.

Since we currently list the docker installation as supported, I'd just add this to our Vagrantfile?
->
```
    d.create_args = ["--cap-add=SYS_ADMIN", "--security-opt=apparmor:unconfined", "--device=/dev/fuse"]
```